docs/application/web/guides/TOC.md

```diff
@@ -172,11 +172,10 @@
 ### [Page Visibility](/application/web/guides/w3c/perf-opt/page.md)
 ### [Timing control for script-based animations](/application/web/guides/w3c/perf-opt/timing-control.md)
 ### [Performance Improvement](/application/web/guides/w3c/perf-opt/performance.md)
-### Web Workers
-#### [Overview](/application/web/guides/w3c/perf-opt/web-workers.md)
-#### [JavaScript Performance Improvement](/application/web/guides/w3c/perf-opt/js-performance-improvement.md)
-#### [JavaScript and CSS Minification](/application/web/guides/w3c/perf-opt/minify-js-css.md)
-#### [jQuery Performance Improvement](/application/web/guides/w3c/perf-opt/jquery-performance-improvement.md)
+### [Web Workers](/application/web/guides/w3c/perf-opt/web-workers.md)
```

Comment:
Question : `Web Workers` and sub pages are on same lever now. Is it to fit in 3 depth?

Reply:
Web workes and children are related in terms of talking about java script performance, but have relationship with a parent and children. After I read their contents, I knew that the below three topics were not descriptions of Web works functionalities.

Moreover, I don't want to create a new TOC.md because not only they're not a parent and children but also the tree depth is more than three.